src/components/ArticlesList.jsx

```diff
@@ -0,0 +1,42 @@
+import { useState, useEffect } from 'react';
+import ArticleCard from './ArticleCard';
+import { getArticles, getUsers } from '../api';
+
+const ArticlesList = () => {
+    const [articles, setArticles] = useState([]);
+//    const [authors, setAuthors] = useState([]);
+    const [isLoading, setIsLoading] = useState(true);
+    const [isError, setIsError] = useState(false);
+
+    useEffect(() => {
+        setIsLoading(true)
+        getArticles()
+        .then((articles) => {
+            setArticles(articles)
+            setIsLoading(false)
+        })
+        .catch((err) => {
+            setIsError(true)
+        })
+    },[])
+
+    if (isLoading) {
+        return <p>Loading...</p>
+    }
+
+    if (isError) {
+        return <p>Sorry, articles not found</p>
```

Comment:
The error doesn't necessarily mean the articles aren't found! For example, it could also mean there's been a server error when making the request to our api. Take another look at the notes on error handling for more info! [[https://l2c.northcoders.com/courses/sd-notes/front-end#sectionId=error-handling,step=intro](https://www.google.com/url?q=https://l2c.northcoders.com/courses/sd-notes/front-end%23sectionId%3Derror-handling,step%3Dintro&sa=D&source=calendar&ust=1731316854091781&usg=AOvVaw115h-Hm_5qLdgYjNy-rBUf)](url)